releasenotes/notes/detect-win-ec2-instance-3e90f6ad7ae03683.yaml

```diff
@@ -0,0 +1,11 @@
+# Each section from every releasenote are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
+  - |
+    Handle Amazon Windows EC2 instances detection which have default hostnames starting with ``EC2AMAZ-`` on Windows.
```

Comment:
let's remove the release note altogether until we can fully predict the impact of enabling the config option